src/utils/ConfigLoader.ts

```diff
@@ -31,19 +31,25 @@ export const getExpressAppConfig = (directory: string) => (app: express.Applicat
   app.use(handler);
 
   app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({ extended: true }));
+  app.use(bodyParser.urlencoded({extended: true}));
   app.use(cookieParser());
 
   app.set('view engine', 'njk');
-    nunjucks.configure([
-      'dist/views',
-      'node_modules/govuk-frontend',
-      'node_modules/govuk-frontend/components',
-    ], {
-      autoescape: true,
-      express: app,
-    });
-
-    app.locals.ROOT_URI = ROOT_URI;
+  nunjucks.configure([
+    'dist/views',
+    'node_modules/govuk-frontend',
+    'node_modules/govuk-frontend/components',
+  ], {
+    autoescape: true,
+    express: app,
+  });
+
+  app.locals.ROOT_URI = ROOT_URI;
+
+  const session: Record<string, any> = {
+    companyNumber: '00345567'
+  };
+
+  app.locals.session = session;
```

Comment:
I think moving the middleware outside of the config loader and into a separate file (like I did for the submission summary:
https://github.com/companieshouse/lfp-appeals-frontend/blob/6ee8e52ccde4abed11ea8df436c505e55b4f5a0e/src/middleware/sessionMocker.ts) 
 is neater, what do you think @geek-soft

Reply:
Makes sense @akaJSaunders

Reply:
i have refactored this :-)